onnxruntime/test/platform/ios/ios_package_test/Podfile

```diff
@@ -4,12 +4,9 @@ target 'ios_package_test' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
 
-  # Pods for ios_package_test
-  pod 'onnxruntime-mobile', :podspec  => './onnxruntime-mobile.podspec'
-
   target 'ios_package_testTests' do
     inherit! :search_paths
-    # Pods for testing
+    pod 'onnxruntime-mobile-c', :podspec  => './onnxruntime-mobile-c.podspec'
   end
```

Comment:
should the pod be renamed to onnxruntime-mobile-c++ or onnxruntime-mobile-cpp if it now needs the C++ compiler?

Reply:
I want to check in this first, then we can rename the package, update objc pod, and related document in another change together

Reply:
ok